src/docs/release/breaking-changes/use-maxLengthEnforcement-instead-of-maxLengthEnforced.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Use maxLengthEnforcement instead of maxLengthEnforced
+description: Introducing the MaxLengthEnforcement enum.
+---
+
+## Summary
+
+_If you don't use the `maxLength` or `maxLengthEnforced` properties in a
+`TextField`, `TextFormField`, or `CupertinoTextField`,
+or you don't care about the truncation behavior of an
+editable text field, you can stop reading._
+
+To control the behavior of `maxLength`
+in the `LengthLimitingTextInputFormatter`,
+use `maxLengthEnforcement` instead of `maxLengthEnforced`, which is deprecated.
+
+## Context
+
+Before [PR 63754][]: Fix TextField crashed with composing and maxLength set,
+the `LengthLimitingTextInputFormatter` always truncated the value in the text
+field, which resulted in engine crashes when the composing range sometimes
+overflowed. With this change, an editable `TextEditingValue` is not truncated
+if only `maxLength` is set, but _is_ truncated after editing is complete.
+
+With [PR 68086][]: Introduce `MaxLengthEnforcement`,
+the truncation behavior is now controlled by passing
+`maxLengthEnforcement` to the `LengthLimitingTextInputFormatter`,
+which is then exposed to the `TextField`, `TextFormField`,
+and `CupertinoTextField` widgets.
+The `maxLengthEnforced` property is now deprecated.
+(For more information, see the docs for [`maxLength`][] and
+[`MaxLengthEnforcement`][].)
+
+The default max length truncation behavior is based on the `TargetPlatform`
+that you're using or running on.
+
+## Description of change
+
+* Added a `MaxLengthEnforcement` enum, which controls all max length
+  limiting behavior.
+* The `maxLengthEnforced` property in editable text field classes is deprecated.
+* A named optional parameter, `maxLengthEnforcement` is added to
+  the `TextField`, `TextFormField`, `CupertinoTextField`,
+  and `LengthLimitingTextInputFormatter` classes.
+* The default behavior for different platforms is updated, including Android,
```

Comment:
Should we raise the `Platforms behavior` part to here, or just add a short description like you suggested?

Reply:
Just thought it would be useful to provide the defaults since it's brought up here, otherwise this line will probably leave people wondering what the defaults are.

Reply:
I'm experiencing watching these long parameters at everywhere 🤣